src/Utils/Decoder.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace MultiversX\Utils;
+
+use Brick\Math\BigInteger;
+
+class Decoder
+{
+    public static function fromBase64ToInt(string $value): int
+    {
+        return hexdec(bin2hex(base64_decode($value)));
+    }
+
+    public static function fromBase64ToBigInteger(string $value): BigInteger
+    {
+        $decodedHex = bin2hex(base64_decode($value));
+
+        return static::bchexdec($decodedHex);
+    }
+
+    public static function bchexdec(string $hex): BigInteger
```

Comment:
Perhaps add more test vectors in `DecoderTest.php`?

Reply:
Added